test/precompile/LVUser/src/LVUser.jl

```diff
@@ -0,0 +1,35 @@
+module LVUser
+
+export filter2davx
+
+using LoopVectorization
+using SnoopPrecompile
+
+function filter2davx!(out::AbstractMatrix, A::AbstractMatrix, kern::AbstractMatrix)
+    @turbo for J in CartesianIndices(out)
+        tmp = zero(eltype(out))
+        for I ∈ CartesianIndices(kern)
+            tmp += A[I + J] * kern[I]
```

Comment:
```suggestion
            tmp += A[I + J - 1] * kern[I]
```
The maximum values of `I` and `J` are `size(kern)` and `size(out)`, so `I+J`'s maximum is `size(kern) .+ size(out) == size(kern) .+ size(A) .- size(kern) .+ 1 == size(A) .+ 1`, hence we get a bounds error on indexing `A`.
The minimum possible value is `2`.

Alternatively, you could use `OffsetArray`s.

This will hopefully fix those segfaults.